pontoon/settings/base.py

```diff
@@ -301,11 +301,16 @@ def _default_from_email():
             log = logging.getLogger(__name__)
             log.error(f"Invalid IP or IP range defined in BLOCKED_IPS: {ip}")
 
+THROTTLE_MAX_COUNT = 100  # A maximum of 100 requests
+THROTTLE_PERIOD = 60  # allowed in 1 minute.
```

Comment:
Looking at today's attach, this might have been too optimistic. 

With multiple IPs creating less than 50 requests in 3–4 minutes Pontoon goes down. Maybe 50 requests over 60s is a more sane value.

Reply:
50/m is easy to hit for legitimate users.

I also see up to 1000 RPS during the latest incident.

I'd start with a conservative number and then tune it.